packages/runtime/src/useCases/transport/relationshipTemplates/CreateOwnRelationshipTemplate.ts

```diff
@@ -68,16 +68,19 @@ export class CreateOwnRelationshipTemplateUseCase extends UseCase<CreateOwnRelat
 
     private async validateRelationshipTemplateContent(content: any) {
         const transformedContent = Serializable.fromUnknown(content);
-        if (!(transformedContent instanceof RelationshipTemplateContent)) return;
+        if (transformedContent instanceof RelationshipTemplateContent) {
+            const validationResult = await this.outgoingRequestsController.canCreate({ content: transformedContent.onNewRelationship });
+            if (validationResult.isError()) return validationResult.error;
 
-        const validationResult = await this.outgoingRequestsController.canCreate({ content: transformedContent.onNewRelationship });
-        if (validationResult.isError()) return validationResult.error;
+            if (transformedContent.onExistingRelationship) {
+                const validationResult = await this.outgoingRequestsController.canCreate({ content: transformedContent.onExistingRelationship });
+                if (validationResult.isError()) return validationResult.error;
+            }
 
-        if (transformedContent.onExistingRelationship) {
-            const validationResult = await this.outgoingRequestsController.canCreate({ content: transformedContent.onExistingRelationship });
-            if (validationResult.isError()) return validationResult.error;
+            return;
+        } else if (transformedContent instanceof ArbitraryRelationshipTemplateContent) {
+            return;
         }
-
-        return;
+        return RuntimeErrors.general.invalidPropertyValue("A relationship template content must be of type RelationshipTemplateContent or ArbitraryRelationshipTemplateContent.");
```

Comment:
How about: "The content of a RelationshipTemplate must be of type RelationshipTemplateContent or ArbitraryRelationshipTemplateContent."?